traffic_portal/app/src/common/modules/table/users/TableUsersController.js

```diff
@@ -28,7 +28,8 @@ var TableUsersController = function(users, $scope, $state, locationUtils) {
         { "name": "Username", "visible": true, "searchable": true },
         { "name": "Email", "visible": true, "searchable": true },
         { "name": "Tenant", "visible": true, "searchable": true },
-        { "name": "Role", "visible": true, "searchable": true }
+        { "name": "Role", "visible": true, "searchable": true },
+        { "name": "Registration Sent", field: "registrationSent", "visible": false }
```

Comment:
this doesn't look right. doesn't look lie the others so i'm guessing it's wrong...

Reply:
There is.. it has been used in other controllers, such as servers, delivery service. etc. So, it is not wrong, just different.

Reply:
yeah, just make it look like lines 28-31 of TableUsersController as TableUsersController uses the old style table.